app/controllers/application_controller.rb

```diff
@@ -2,7 +2,7 @@
 class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  protect_from_forgery prepend: true, with: :exception
```

Comment:
This as well was something that I was told to do for Rails 5 projects with Devise